VideoAPI/VideoApi.Contract/Responses/ExpiredConferencesResponse.cs

```diff
@@ -14,5 +14,10 @@ public class ExpiredConferencesResponse
         /// The current conference status
         /// </summary>
         public ConferenceState CurrentStatus { get; set; }
+
+        /// <summary>
+        /// The hearing Id
+        /// </summary>
+        public Guid HearingId { get; set; }
```

Comment:
on video api I think the references to HearingId are HearingRefId

Reply:
I think it's should not match Conference type, it just meaning that we need hearingId in return

Reply:
it's a different type and has field HearingId, why it should not be?

Reply:
Ok lets leave it like that then.